### PR TITLE
add trivy-image module

### PR DESCRIPTION
**Pull request by JonathanSerafini**

### CODE REVIEW

File: scanners/boostsecurityio/trivy-image/module.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+api_version: 1.0
+
+
+id: boostsecurity-trivy-image
```
Comment (fproulx-boostsecurity) [inline scanners/boostsecurityio/trivy-image/module.yaml:4]:
```suggestion
id: boostsecurity/trivy-image
```

File: scanners/boostsecurityio/trivy-image/module.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+name: Boostsecurity Trivy Image Scanner
+namespace: boostsecurity/trivy-image
+
+
+config:
+  require_full_repo: true
+  support_diff_scan: true
```
Comment (fproulx-boostsecurity) [inline scanners/boostsecurityio/trivy-image/module.yaml:11]: Does this actually support diff scans ?

**Review by fproulx-boostsecurity — CHANGES_REQUESTED** [review]
`identifier` should align with `namespace` as we agreed at standup

File: scanners/boostsecurityio/trivy-image/module.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+api_version: 1.0
+
+
+id: boostsecurity-trivy-image
+name: Boostsecurity Trivy Image Scanner
+namespace: boostsecurity/trivy-image
```
Comment (fproulx-boostsecurity) [inline scanners/boostsecurityio/trivy-image/module.yaml:6]:
```suggestion
namespace: boostsecurityio/trivy-image
```

File: scanners/boostsecurityio/trivy-image/module.yaml (hunk 2 above)
Comment (JonathanSerafini) [reply scanners/boostsecurityio/trivy-image/module.yaml:11]: don't see why it wouldn't

**Comment by JonathanSerafini** [comment]
rebased, something seems to have changed in the action

**Review by fproulx-boostsecurity — APPROVED** [review]

**Pull request merged**